MELA/python/mela_binding.cpp

```diff
@@ -215,6 +217,29 @@ SimpleParticleCollection_t collection_initializer(py::list listOfParticles){
     return collection;
 }
 
+void setInputEvent(Mela& mela, SimpleParticleCollection_t* daughters, SimpleParticleCollection_t* associated, SimpleParticleCollection_t* mothers, bool isgen){
+    if((daughters) && (daughters->size() == 0)){
+        daughters = 0;
+    }
+    if ((associated) && (associated->size() == 0)){
+        associated = 0;
+    }
+    if ((mothers) && (mothers->size() == 0)){
+        mothers = 0;
+    }
```

Comment:
Style comments:
- `(daughters) &&` is the same thing as `daughters &&`; no need for parentheses. Same for the `size() == 0` checks.
- Try to use `nullptr` instead of `0` for pointers. They end up being the same thing, but it allows better interpretability.
- Instead of `size() == 0`, it is preferable to use `empty()`.

Question: Does binding work if I pass `None` for, say, `mothers`. What about the default of `isgen`, which is present in the `Mela` class (https://github.com/JHUGen/JHUGenMELA/blob/master/MELA/interface/Mela.h#L67)?

Reply:
It does work for `None`! Good catch. The wrapper is now not necessary, one can pass None in. I can set the default parameters as well.